src/Landing/html/landing-page.html

```diff
@@ -0,0 +1,88 @@
+                </div>
+                <div class="nav-item-profile">
+                    <img class="drop-shadow profile-avatar" src="./assets/img_avatar.png" alt="Avatar" >
+                </div>
+            </div>
+        </div>
+        <div class="landing-body">
+            <div class="section section-one">
+                <img class="drop-shadow landing-sec-1-img" src = "./assets/landing_1.svg" alt="landing_demo"/>
+                <div class="landing-sec-1-heading font-white">
+                    Welcome to <br>the NextGen theater
+                </div>
+                <div class="landing-sec-1-heading-divider"></div>
+                <div class="landing-sec-1-sub-heading font-white">
+                    Host video parties across the major streaming services, and <br>
+                    connect with your friends and family watching your favourite shows.
+                </div>
+                <div class="landing-sec-1-button-container">
+                    <div style="flex:1">
+                        <div class="drop-shadow watch-party-button font-white" onclick="alert('Watch Party!')">Watch Party</div>
+                    </div>
+                    <div style="flex:1">
+                        <div class="drop-shadow learn-more-button font-white" onclick="alert('Learn More!')">Learn More</div>
+                    </div>
+                </div>
+            </div>
+            <div class="section section-one">
+                <div class="landing-multi-image-container">
+                    <img class="drop-shadow landing-sec-2-img" src = "./assets/landing_2.svg" alt="landing_demo"/>
+                    <img class="drop-shadow landing-sec-3-img" src = "./assets/landing_3.svg" alt="landing_demo"/>
+                </div>
+                <div class="landing-sec-2-heading font-white">
+                    CREATE YOUR THEATER, OWN IT
+                </div>
+                <div class="landing-sec-1-heading-divider"></div>
+                <div class="landing-sec-1-sub-heading font-white">
+                    Host a video party for up to a thirty guests, or have a virtual movie night with friends. Hop on the website <br> and take charge while your show is synced for everyone.
+                </div>
+                <div class="landing-sec-1-button-container">
+                    <div class="drop-shadow learn-more-button font-white" onclick="alert('Learn More!')">Learn More</div>
+                </div>
+            </div>
+            <div class="section section-one">
+                <div class="landing-multi-image-container">
+                    <img class="drop-shadow landing-sec-4-img" src = "./assets/landing_4.svg" alt="landing_demo"/>
+                    <img class="drop-shadow landing-sec-5-img" src = "./assets/landing_5.svg" alt="landing_demo"/>
+                </div>
+                <div class="landing-sec-2-heading font-white">
+                    ENJOY TOGETHER, WHEREVER
```

Comment:
`text-transform: uppercase` would be better, this will be painful for users relying on screen-readers